nautobot/dcim/tests/test_migrations.py

```diff
@@ -967,3 +992,27 @@ def test_region_and_site_data_migration(self):
             loc_clusters = self.cluster.objects.filter(name__in=["Cluster 3", "Cluster 4"])
             self.assertEqual(loc_clusters[0].location.name, "Test Location 0")
             self.assertEqual(loc_clusters[1].location.name, "Test Location 0")
+
+        with self.subTest("Testing Users app model migration"):
+            region_permissions = self.object_permission.objects.filter(
+                name__in=["Test Region Permission 1", "Test Region Permission 2"]
+            )
+            self.assertEqual(
```

Comment:
You could make this more concise by using [assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual) and dropping `sorted`. Not blocking

Reply:
Yeah, I do want to check the actual content of the many-to-many field just to make sure it is correct.

Reply:
`assertCountEqual` does check that the content of the iterables matches it just ignores the sort order. I don't know why they named it `assertCountEqual`